samples/core/Benchmarks/Benchmarks.csproj

```diff
@@ -1,12 +1,14 @@
-<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
-  </PropertyGroup>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
```

Comment:
2 space indentation in csproj files please.

Reply:
I suppose that an authoritative .editorconfig would have enforced this to give consistent result that we can all celebrate [hooray!]. Anyway, as PR Contributor I always tick the "Collaborators can red-ink and re-hack-at-will" [ok it only nearly says that] so go for pasteurised merge and then we can all move on

Reply:
Am also removing the package reference to SqlClient. While it's indeed a good idea to use the latest SqlClient, I'd really rather keep our samples as simple as possible etc.